lang/src/main/kotlin/org/partiql/lang/CompilerPipeline.kt

```diff
@@ -21,7 +21,6 @@ import org.partiql.lang.eval.Bindings
 import org.partiql.lang.eval.CompileOptions
 import org.partiql.lang.eval.EvaluatingCompiler
 import org.partiql.lang.eval.ExprFunction
-import org.partiql.lang.eval.ExprValueFactory
```

Comment:
I mean to say, why was this import removed and ExprValueFactory fully qualified? `Suppress("DEPRECATION")` seems to work fine without needing to fully qualify ExprValueFactory

Reply:
Sorry for that email-response.  

So, when this file-level import is present, it gives a warning that it imports a deprecated class.  I do not know how to suppress this warning:  `@Suppress("DEPRECATION")` is not allowed here.  Hence, fully-qualified in all places where the deprecated class needs to be mentioned.